term/term_unix.go

```diff
@@ -21,7 +20,9 @@ func IsTerminal(f *os.File) bool {
 }
 
 func IsATTY(f *os.File) bool {
-	return termutil.Isatty(f.Fd())
+	stat, _ := f.Stat()
```

Comment:
I don't think it's OK to ignore the error here. Maybe return false in such cases? I have the feeling that it's what is currently done in the dependency: 

https://github.com/andrew-d/go-termutil/blob/009166a695a2f516c749a26b4ac1f183d89aa336/isatty_nix.go#L19